sdk/test-utils/perfstress/test/nodeFetch.spec.ts

```diff
@@ -23,8 +23,8 @@ export class NodeFetchTest extends PerfStressTest<NodeFetchOptions> {
       description: "Required option",
       shortName: "u",
       longName: "url",
-      defaultValue: "http://bing.com",
-      value: "http://bing.com"
+      defaultValue: "http://www.example.org",
+      value: "http://www.example.org"
```

Comment:
`http://bing.com` redirects to `https://bing.com` which can cause issues.  Changed default to `http://www.example.org` which does not redirect to HTTPS.